tests/spread/core24/appstream-desktop/desktop/expected_snap.yaml

```diff
@@ -22,7 +22,7 @@ apps:
     command: usr/bin/appstream-desktop
     common-id: io.snapcraft.appstream
 confinement: strict
-grade: devel
+grade: stable
```

Comment:
re @lengau: I don't think there was a discrepancy in the tests, the only places where I "kept" the grade is in these `expected_snap.yaml` files, not the original `snapcraft.yaml` files.